### ISSUE TITLE
truncate or remove 'tags' block from site search results

### ISSUE THREAD

**Issue by rahaeli** [dreamwidth/dw-free#2594]
https://www.dreamwidth.org/search includes the tags an author tagged their entry with, which has been reported as being a pain to scroll past if the author is a compulsive overtagger ala Tumblr/AO3 tagging conventions (especially on mobile). If it's not computationally expensive, let's truncate to only show the first 5 tags on an entry; if it is, which I suspect it might be, let's just remove showing the tags at all, since they're not generally super-informative in a search context.

**Comment by kareila** [comment]
There is one context in which I find it useful to see tags in site search, which is locating spam accounts. SEO spam accounts tend to use SEO keywords as entry tags.

**Comment by rahaeli** [comment]
Yeah, that was the only thing that made me say truncate if we can other than just removing. It is useful for that!
